driver-portal/src/ClientApp/src/app/letters-to-driver/letters-to-driver.component.ts

```diff
@@ -36,7 +36,9 @@ import { CaseTypeComponent, LetterTopicComponent } from '@shared/core-ui';
     ],
 })
 export class LettersToDriverComponent implements OnInit {
-  constructor(private caseManagementService: CaseManagementService) {}
+  constructor(private caseManagementService: CaseManagementService,  
+    private loginService: LoginService) 
```

Comment:
this is userService in partner portal. You can also use userService here

Reply:
We don't need loginservice in this layer as driver id is not being used. I will change that in partner portal as well.